server/templates/static/homepage.html

```diff
@@ -82,9 +82,8 @@ <h2>Explore the data</h2>
       <div class="col-12">
         <p>
           Explore the data using one of our visualization tools (<a href="{{ url_for('tools.scatter') }}">Scatter</a>, <a href="{{ url_for('tools.map') }}">Map</a>,
-          <a href="{{ url_for('tools.timeline') }}">Timeline</a>) or use the <a href="https://docs.datacommons.org/api">APIs</a> via Python notebooks or build your own
-          application. Explore the thousands of variables in the <a href="{{ url_for('tools.stat_var') }}">Data Dictionary
-          Exploration Tool</a>. Learn how you can <a target="blank" href="https://docs.datacommons.org/contributing">contribute to the project</a>.
+          <a href="{{ url_for('tools.timeline') }}">Timeline</a>), the <a href="{{ url_for('place.place') }}">Place Explorer</a>, or use the <a href="https://docs.datacommons.org/api">APIs</a> via Python notebooks or build your own
+          REST based application. Explore the thousands of variables in the <a href="{{ url_for('tools.stat_var') }}">Statistical Variable Explorer</a>. Contribute to the project on <a target="blank" href="https://github.com/datacommonsorg">Github {{ external_icon() }}</a>.
```

Comment:
Curious if we had wanted to change the contributing link back to github because I remember changing the contributing link to the doc site link was a very recent task

Reply:
thanks for that catch! i missed it (this was a pending change from before that update)